doc/classes/ProjectSettings.xml

```diff
@@ -1941,6 +1955,44 @@
 		<member name="rendering/reflections/sky_reflections/texture_array_reflections.mobile" type="bool" setter="" getter="" default="false">
 			Lower-end override for [member rendering/reflections/sky_reflections/texture_array_reflections] on mobile devices, due to performance concerns or driver support.
 		</member>
+		<member name="rendering/renderer/renderer_name" type="String" setter="" getter="" default="&quot;forward_plus&quot;">
```

Comment:
I think we should use "rendering_method" rather than "renderer_name". This should make it less ambiguous with "rendering_driver". Maybe we shuld eventually also rename the RenderScene class to RenderMethod given it will now become pretty much that.